src/core/lib/security/transport/security_connector.c

```diff
@@ -42,13 +42,19 @@
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>
 
+#include "src/core/ext/client_channel/client_channel_factory.h"
+#include "src/core/ext/client_channel/uri_parser.h"
+#include "src/core/ext/lb_policy/grpclb/grpclb.h"
```

Comment:
I think that *all* security connectors will have to use that in their `check_peer` and `check_call_host` methods since it is needed to get the name of the load balancer connection (see below for SSL). It would actually be beneficial to be have a common util to extract this name (or the name to check in general which would be the load balancer name or the original target name).